library/modules/Items.cpp

```diff
@@ -2133,14 +2133,17 @@ int32_t Items::createItem(df::item_type item_type, int16_t item_subtype, int16_t
     delete prod;
 
     DEBUG(items).print("produced %zd items\n", out_items.size());
```

Comment:
since we're using the passed-in vector directory, should we clear it before passing it to `produce`?

edit: yes we totally should since we're using the size to determine success for the return value